### DISCUSSION TITLE
[Nightmare Zone] AFK Improvements

### DISCUSSION THREAD

**Discussion by pairofcrocs** [runelite/runelite#17758] [Ideas]
I've been doing a lot of 6 hour AFKing recently and noticed the need for some helpful features that would make this plugin even better than it already is. All of these requests would help from having to remote into a computer to check the status of simple things.

***Discord Webhook Support***

The big two would be a **Login notification**, as well as a **log out notification**.

Ideally, a timer would be sent so you know how much time is left before you get auto logged out. This can be done by sending                       `<t:unixtimestamphere:R>` where `unixtimestamphere` is 6 hours from the current time.
        
**Guthans Health Notifications**

Guthans is very popular to train with but the armour degrades and if you don't repair it, it breaks.
I've had many session where I die because I forgot the repair the armour, a simple notifcation could be sent to the webhook so people know when it's time to repair.

**Comment by iProdigy** [comment]
for your first feature request, [dink](https://runelite.net/plugin-hub/show/dink) can already help:

* it can send a notification on login (`Advanced > Custom Metadata Handler`)
* it can send a notification upon receiving the nerdlog warning chat message (`Custom Chat Messages > Enable Chat Notifications` and `Custom Chat Messages > Message Filters`)
